darts/utils/multioutput.py

```diff
@@ -1,9 +1,8 @@
-from joblib import Parallel
 from sklearn.base import is_classifier
 from sklearn.multioutput import MultiOutputRegressor as sk_MultiOutputRegressor
 from sklearn.multioutput import _fit_estimator
-from sklearn.utils.fixes import delayed
 from sklearn.utils.multiclass import check_classification_targets
+from sklearn.utils.parallel import Parallel, delayed
```

Comment:
Hope that it does not break with python 3.7 which support is being dropped left and right... On scikit-learn github repository, they indicate "scikit-learn 1.0 and later require Python 3.7 or newer. scikit-learn 1.1 and later require Python 3.8 or newer.". Could not find in the log when did the parallel module change...

Reply:
Yes that's true, let's check later on in merge workflow.